src/mixins/experiment.js

```diff
@@ -11,7 +11,7 @@ export default function experimentsMixin(
         const { name, $activeVariants } = this.$exp;
         if (
           this.$exp.isEligible &&
-          !this.$exp.isEligible({ route: this.$route })
+          !this.$exp.isEligible({ route: this.getRouteBaseName(this.$route) })
```

Comment:
should pass `getRouteBaseName` into `isEligible()` as route is an object with query and params